web/sass/_project.scss

```diff
@@ -1262,3 +1262,5 @@ form.signup button[type="submit"] {
   font-size: 0.875em;
   padding: 1px;
 }
+
+@import "_footer.scss";
```

Comment:
Ah, actually I think _footer is missing, or has been included at end of mysoc-footer by mistake.